### ISSUE TITLE
Damage only actions not sending properly to chat

### ISSUE THREAD

**Issue by Tabletop-RPG-Music** [giffyglyph/foundry-5e-monster-maker#29]
If you make an action that is just damage (without an attack roll / saving throw), it just sends the name of the action to chat but not the damage. (Not sure if there's another way to make a macro for damage?)

**Comment by Tabletop-RPG-Music** [comment]
I think this might be a compatibility issue with midi-qol

**Comment by giffyglyph** [comment]
Looks like this is a midi-qol compatibility issue. The problem stems from midi-qol/utils.js:

```
export function itemHasDamage(item) {
    return item?.data.data.actionType !== "" && item?.hasDamage;
}
```

In FMM, damage can be set when no actionType is specified. Mqol is being too zealous in its checking here. Have now added the "utility" attack type to support damage-only prompts.

**Comment by giffyglyph** [comment]
Fixed in v1.0.0. Can now select "utility" action type in scaling actions.

**Comment by Tabletop-RPG-Music** [comment]
Thanks for this. 

It might be something that needs to be changed by the midi-qol developer rather than on your end, but I've noticed that with this latest update, midi-qol isn't applying any damage rolls, regardless of whether they're Utility or part of an attack roll / saving throw. 

For example, clicking the damage button rolls the dice (if you have dice-so-nice installed) but doesn't show up any results.
![image](https://user-images.githubusercontent.com/16239913/126158050-5a43acd5-c948-4d09-ac39-3bb637f2f31a.png)

No pressure to fix this as I know it's probably a niche use case (I just love both modules and would love for them to work together)
